artifacts/centos/centos.go

```diff
@@ -147,11 +147,11 @@ func (c *centos) Tests() []api.ArtifactTest {
 }
 
 // New accepts CentOS 7 and 8 versions. Example patterns are 7-2111, 7-2009, 8.3, 8.4, ...
-func New(release string, additionalLabels map[string]string) *centos {
+func New(release, arch string, additionalLabels map[string]string) *centos {
```

Comment:
Can you make these take a slice of architectures to build instead? This way we can create the manifest containing all the possible architectures down the line in the `build` pkg.

Reply:
Do you know if CDI will support it, or already supports it?

Reply:
IIUC it should transparent to CDI. It relies on the node to pull the correct image. It will just provide the ref and the CRI on the node will resolve the correct architecture from the image's manifest.